core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/dot-edit-content-aside.component.scss

```diff
@@ -0,0 +1,20 @@
+@use "variables" as *;
+
+.edit-content-aside--header {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: $spacing-1;
+}
+
+.edit-content-aside--item {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    align-items: center;
+    height: $spacing-5;
+    margin-bottom: $spacing-1;
+
+    p {
+        color: $color-palette-gray-800;
```

Comment:
We need to talk to @melissarojas-dotcms about this, doesn't make sense to me have a color so close to the body color just for this...